src/chat/takeover.js

```diff
@@ -359,4 +359,12 @@ var takeover = module.exports = function() {
             }
         }
     });
+
+    $('.tse-content').on('click', '.chat-line', function() {
```

Comment:
I added it to the line rather than the name because when you click names it brings up the popup box where you can message that specific user etc.

If you're still fine with adding it just to names then I'll go ahead and change it to that.